pkg/pillar/cmd/zedkube/zedkube.go

```diff
@@ -6,12 +6,16 @@
 package zedkube
 
 import (
+	"net/http"
+	"strings"
 	"sync"
 	"time"
 
+	// "github.com/lf-edge/eve/pkg/newlog/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.22.5.darwin-amd64/src/strings"
```

Comment:
Accidentally added?

Reply:
was used for something else during dev, will remove.